src/OrchardCore.Modules/OrchardCore.Tenants/ViewModels/TenantViewModel.cs

```diff
@@ -22,7 +22,7 @@ public class TenantViewModel
 
         public string RecipeName { get; set; }
 
-        public string FeatureProfile { get; set; }
+        public string[] FeatureProfile { get; set; }
```

Comment:
Rename the view model at least, it's not breaking anything

Reply:
I renamed the property like this `public string[] FeatureProfile { get; set; }` is this enough @sebastienros?